src/net_processing.cpp

```diff
@@ -4335,15 +4335,15 @@ void PeerManagerImpl::CheckForStaleTipAndEvictPeers()
 {
     LOCK(cs_main);
 
-    int64_t time_in_seconds = GetTime();
+    auto time_in_seconds{GetTime<std::chrono::seconds>()};
```

Comment:
Maybe rename to `now` for brevity?

Reply:
Makes sense. I am going to address it in the next update of PR.

Reply:
Addressed in 1a1fafa4fdcbe7b0d1ef8df466a71bd3e65195d3